src/main/java/org/broadinstitute/hellbender/tools/walkers/genotyper/afcalc/AlleleFrequencyCalculator.java

```diff
@@ -104,7 +104,7 @@ public AFCalculationResult getLog10PNonRef(final VariantContext vc, final int de
                 final double log10GenotypePosterior = log10GenotypePosteriors[genotype];
                 glCalc.genotypeAlleleCountsAt(genotype).forEachAlleleIndexAndCount((alleleIndex, count) ->
                         log10ProbabilityOfNonZeroAltAlleles[alleleIndex] =
-                                MathUtils.approximateLog10SumLog10(log10ProbabilityOfNonZeroAltAlleles[alleleIndex], log10GenotypePosterior));
+                                MathUtils.log10SumLog10(log10ProbabilityOfNonZeroAltAlleles[alleleIndex], log10GenotypePosterior));
```

Comment:
@vruano I already made the change and the performance is fine, so if you don't mind I'll keep it *un*approximate.  (I would say "exact" but that's an overloaded adjective here).